README.md

```diff
@@ -89,7 +89,7 @@ export NAMESPACE=conjur
 Configure the container images:
 
 ```shell
-export TAG_VERSION=1.3.4
+export TAG_VERSION=1.3
```

Comment:
I think pinning to full semver version might be better than to the minor version since you won't be able to know what version of components you're pulling/writing.

Reply:
Okay, I switched this to the full semver `1.4.0` in anticipation of us making a release `1.4.0` and then publishing it to Google Cloud Marketplace.